gwt-material/src/main/java/gwt/material/design/client/ui/MaterialCollapsible.java

```diff
@@ -113,6 +119,37 @@ public MaterialCollapsible() {
         // Items need to be added after the widget has loaded to avoid
         // premature configuration issues.
         enableFeature(Feature.ONLOAD_ADD_QUEUE, true);
+
+        // initialize the mutation observer responsible for firing the collapse and expand events
+        observer = new MutationObserver((records, o) -> onMutation(records));
+    }
+
+    /**
+     * @param records
+     * @return
+     */
+    private Object onMutation(MutationRecord[] records) {
+        for (MutationRecord r : records) {
+            System.out.println(r.target.nodeName + " " + r.attributeName + ": " + r.oldValue + " type: " + r.type);
```

Comment:
Maybe this should be removed :smile:

Reply:
Yup agreed we must remove any debug logs.

Reply:
ups, missed that one. sorry.